spec/system/accessibility_spec.rb

```diff
@@ -105,7 +103,7 @@
       click_on "Submit"
     end
 
-    it "has no AXE-detectable accessibility issues" do
+    it "has no AXE-detectable accessibility issues", :slow do
```

Comment:
question - how did you find out which tests took the longest? I can't see where I would see that information when running the test's locally or viewing on CircleCI.

Reply:
`bundle exec rspec -p` the `-p` is shorthand for `--profile`
and it gives you the 10 slowest running tests

I split the tests out into 2 separate tests and ran them with the profile flag.